OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -912,7 +1022,10 @@ function Remove-WorkerPoolRegistration
         [Parameter()]
         [PSCredential]$TentacleServiceCredential,
         [Parameter(Mandatory = $true)]
-        [string]$name
+        [string]$name,
+        [Parameter()]
```

Comment:
Make the param mandatory, and pull the param name onto the same line as the type